tests/commands/new/test_build_context.py

```diff
@@ -1235,6 +1243,14 @@ def main():
 requires = [
     "toga-android~=0.4.0",
 ]
+
+base_theme = "Theme.MaterialComponents.Light.DarkActionBar"
+
+build_gradle_dependencies = [
+    "com.google.android.material:material:1.11.0",
```

Comment:
The dependency list gets a lot simpler under material, because appcompat and constraintlayout are both dependencies of material.